bin/install-if-deps-outdated.js

```diff
@@ -54,6 +54,7 @@ function install() {
 	const installResult = spawnSync( 'npm', [ 'ci' ], {
 		shell: true,
 		stdio: 'inherit',
+		env: Object.assign( { PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: 'true' }, process.env ),
```

Comment:
we're on node 12, you can just write `{ PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: 'true', ...process.env }`

Reply:
👍 231c4844b23ae53e8f7d02ca498d8fe938094d72